docs/topics/i18n/timezones.txt

```diff
@@ -311,10 +312,9 @@ time zone is unset, the default time zone applies.
 get_current_timezone
 ~~~~~~~~~~~~~~~~~~~~
 
-When the ``django.template.context_processors.tz`` context processor is
-enabled -- by default, it is -- each :class:`~django.template.RequestContext`
-contains a ``TIME_ZONE`` variable that provides the name of the current time
-zone.
+If you enable the ``django.template.context_processors.tz`` context processor,
```

Comment:
I'd like if we provided some guidance on whether to prefer the context processor or the template tag (for all cases like this).